docs/reference/index.md

```diff
@@ -20,6 +20,7 @@ This section contains the reference documentation for the public API of the proj
 - [Automatic Speech Recognition (ASR) Models](./models/asr.md) - Models for working with automatic speech recognition (ASR) models.
 - [Caption Models](./models/captions.md) - Models for working with captions.
 - [Chat Models](./models/chat.md) - Models for working with chat models.
+- [Image Chat Models](./models/image_chat.md) - Models for working with chat containing text and images as input.
```

Comment:
Can you rewrite the description to be more clear? Maybe mention visual-language models.